acq4/devices/Stage/Stage.py

```diff
@@ -572,12 +572,24 @@ def goHome(self, speed='fast'):
     def setHomePosition(self, pos=None):
         """Set the home position in global coordinates.
         """
+        self.setStoredLocation('home', pos)
+
+    def getStoredLocation(self, name):
+        return self.readConfigFile('stored_locations').get(name, None)
+
+    def setStoredLocation(self, name: str, pos=None):
         if pos is None:
             pos = self.globalPosition()
         locations = self.readConfigFile('stored_locations')
-        locations['home'] = list(pos)
+        locations[name] = list(pos)
         self.writeConfigFile(locations, 'stored_locations')
 
+    def clearStoredLocation(self, name):
```

Comment:
oh, all of this refactor doesn't get used anymore. it's probably fine to include, but not necessary.